script.js

```diff
@@ -0,0 +1,69 @@
+let developers = [
+    {
+        "name": "Carolin Bloeme",
+        "id": "carolin-bloeme"
+    },
+    {
+        "name": "Sun Jun",
+        "id": "sun-jun"
+    },
+    {
+        "name": "Song Bao",
+        "id": "song-bao"
+    },
+    {
+        "name": "Olivia Arribas",
+        "id": "olivia-arribas"
+    },
+    {
+        "name": "Samah abu laymun ",
+        "id": "samah-abu-laymun"
+    }
+]
+
+let ul = document.getElementById('developers');
+let li = "";
+for (let i = 0; i < developers.length; i++) {
+    li +=
+        `
+  <div class="list-item-wrapper">
+  <li
+  class="list-group-item d-flex justify-content-between align-items-center"
```

Comment:
I would suggest to use your own classes and not Bootstrap.
This is will let you focus more on CSS and let the class property simpler. I understand it is easier for you, but for me, I don't like using this kind of frameworks (It is a personal taste, so you may disagree with me). This is because you will have to remember all the names you want to use or keep searching for them instead of directly write them.
Another thing that you sometimes need to add a class while you can add a property in the class list.

Again, I'm not forcing you not to use it, just saying that maybe in this training at least, it is better not to use it just to make sure everything is understood